packages/cashier/src/Stores/Cashier/on-ramp-store.js

```diff
@@ -189,26 +188,28 @@ export default class OnRampStore extends BaseStore {
 
         const deposit_address_interval = setInterval(() => getDepositAddressFromApi, 3000);
         const getDepositAddressFromApi = () => {
-            WS.authorized.cashier('deposit', { provider: 'crypto', type: 'api' }).then(response => {
-                let should_clear_interval = false;
+            this.root_store.modules.cashier.WS.authorized
```

Comment:
I would recommend to use `WS` from `Services/ws-methods.js`  since `WS` doesn't belong to a store

Reply:
The way we initiate the WebSocket has changed. Using it from `Services/ws-methods.js` creates unnecessary duplicates and complexity across the packages. And, since we are using components outside this package, we can't initiate/passdown WS only when `packages/cashier/src/App.jsx` is loaded. the exposed components of the Cashier are store-connected components that rely on WS.